README.md

```diff
@@ -19,20 +19,38 @@ Troupon is a Groupon clone. Users are able to buy products and services at a dis
 ### <a name="installation"></a>Installation
 - Clone repo: `$ git clone https://github.com/andela/troupon.git`
 - Checkout to branch develop: `$ git checkout develop`
-- Create a .env.py file in your root directory. In it put the following key - value pairs:
+- Create a .env.yml file in your root directory. In it put the following key - value pairs:
 	```cmd
-	DB_USER = "your_database_username_here"
-	DB_PASSWORD = "your_database_password_here"
-	MAILGUN_URL = "your_mailgun_url_here"
-	MAILGUN_USERNAME = "your_mailgun_username_here"
-	MAILGUN_PASSWORD = "your_mailgun_password_here"
-	SECRET_KEY = "abcdefghijklmnop"
-	```
+    sendgrid_apikey:
+      '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'
```

Comment:
Authentication Credentials like this should **NOT** be exposed in the repository. They should be stored instead, as environment variables on your local machine.

Please use descriptive placeholders for these values. e.g 

```
DB_USER = "your_database_username_here"
```